spec/nio/selectables/udp_socket_spec.rb

```diff
@@ -22,7 +22,17 @@
   end
 
   let :writable_subject do
-    pending "come up with a writable UDPSocket example"
```

Comment:
The problem is on Travis, your example manages to create a `writable_subject`, but then the subsequent tests are failing.

I'm not sure why they're failing but it's not entirely unexpected considering this was just pending before. It was likely pending because of sporadic CI failures.